Dockerfile

```diff
@@ -27,10 +28,38 @@ ONBUILD ARG PGUSER=odoo
 ONBUILD ARG PGPASSWORD=odoopassword
 ONBUILD ARG PGHOST=db
 ONBUILD ARG PGDATABASE=prod
-ONBUILD ENV PGUSER="$PGUSER" \
+ONBUILD ARG ODOO_MAX_CRON_THREADS=1
+ONBUILD ARG ODOO_WORKERS=0
+ONBUILD ARG ODOO_LIMIT_TIME_REAL=120
+ONBUILD ARG ODOO_LIMIT_TIME_CPU=60
+ONBUILD ARG ODOO_LIMIT_MEMORY_HARD=805306368
+ONBUILD ARG ODOO_LIMIT_MEMORY_SOFT=671088640
+ONBUILD ARG ODOO_LIMIT_REQUEST=8192
+ONBUILD ARG ODOO_LIMIT_TIME_REAL_CRON=1
+ONBUILD ARG ODOO_DB_FILTER=
+ONBUILD ARG ODOO_LIST_DB=1
+ONBUILD ARG ODOO_LOG_LEVEL=info
```

Comment:
@pedrobaeza You can see the defaults he's adding here.

The problem is that for performance there are no *sane* defaults for everyone. If you apply a 32 core's sane defaults to a 4 core machine, it would get exhausted.

My original idea was to have some sane disabled defaults in config files that you could symlink in your `custom/conf.d` directory.

Imagine something like `/opt/odoo/common/disabled-conf.d/` with files like:

- 1vcpu.conf
- 2vcpu.conf
- 4vcpu.conf
- 8vcpu.conf (etc.)
- 4gbram.conf
- 8gbram.conf
- 32gbram.conf (etc.)

Similar to apache's `sites-available` vs `sites-enabled`.

So you can add those with `ln -s /opt/odoo/common/disabled-conf.d/{8vcpu,32gbram}.conf custom/conf.d` and have a community-maintained sane default for your machine. Or we could use an env var for that if it seems more convenient.